csharp-intermediate-topics/RenameFilesWithCSharp/RenameFilesWithCSharp/BatchRenameFiles.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace RenameFilesWithCSharp;
+
+public class BatchRenameFiles
+{
+    public static void RenamePhotosWithDateTime(string directoryPath)
+    {
+        // An array of allowed file extensions.
```

Comment:
No need to comment every single line of code in the class, especially if it is straightforward like this one. Too many comments usually act as noise and can distract the readers. Thus, if the code is clean and self-explanatory, you can remove all the comments.